src/fragments/start/getting-started/ionic/setup.mdx

```diff
@@ -59,8 +51,22 @@ When you initialize a new Amplify project, a few things happen:
 
 ## Install Amplify libraries
 
-In addition to `aws-amplify` core, install the Angular Ionic modules which provide a service provider, helpers, and components.
+Make sure to install the latest version of the `aws-amplify` core and the `@aws-amplify/ui-angular` library.
 
 ```sh
-$ npm install aws-amplify @aws-amplify/ui-angular@1.x.x
+$ npm install aws-amplify @aws-amplify/ui-angular
 ```
+
+The `@aws-amplify/ui-angular` package is a set of Angular components and an Angular provider which helps integrate your application with the `aws-amplify` library.
+
+<Callout warning={true}>
```

Comment:
Instead of listing all the unusual Angular issues they might have, I just linked to the troubleshooting guide here.